model/distro/distro_test.go

```diff
@@ -428,3 +429,31 @@ func TestGetResolvedPlannerSettings(t *testing.T) {
 	assert.EqualValues(t, 0, resolved2.TimeInQueueFactor)
 	assert.EqualValues(t, 0, resolved2.ExpectedRuntimeFactor)
 }
+
+func TestAddPermissions(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(db.ClearCollections(user.Collection, Collection, evergreen.ScopeCollection, evergreen.RoleCollection))
+	_ = evergreen.GetEnvironment().DB().RunCommand(nil, map[string]string{"create": evergreen.ScopeCollection})
```

Comment:
you get the environment twice here, which might be weird. does it make sense to get a testing environment from the testutil package?